lab_4.py

```diff
@@ -0,0 +1,54 @@
+class Helicopter:
+    # Опис вертольота
+        def __init__(self, passengers=9, name="fIGHT", max_speed=269):
+        #Властивості вертольота
+            self.__passengers = passengers
+            self.__name = name
+            self.__max_speed = max_speed
+    
+        #Метод доступу до приватних полів
+        # def get_passengers(self):
+        #     return self.__passengers
+    
+        # def set_passengers(self, passengers):
+        #     self.__passengers = passengers
+        @property
+        def passengers(self):
+            return self.__passengers
+        
+        def passengers (self, passengers):
+            self.__passengers = passengers
+
+        def get_name(self):
```

Comment:
These methods are not needed if you're already using properties. You can replace these getter and setter methods with @property decorators, which is a cleaner and more Pythonic way to handle attribute access